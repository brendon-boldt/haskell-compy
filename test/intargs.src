with iA and iB and iC, pA does `show iA. show iB + iC.'.
with iA and iC, pB does `show iA. show iC.'.
given iA is 1 and iB is 2 and iC is 3, do pA.
given iA is 9 and iC is 88, do pB.
given iA is 2 and iB is 3 and iC is 4, do pA.

0.
