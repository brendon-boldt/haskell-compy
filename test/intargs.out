1
5
9
88
2
7
